tools/migrate/main.go

```diff
@@ -316,10 +317,41 @@ func migrate(srcDir, dstDir string, rehash bool) error {
 		}
 	}
 
+	// Tread TiUP as a component
+	tiup := v0manifest.ComponentInfo{
+		Name: "tiup",
+		Desc: "Components manager for TiDB ecosystem",
+	}
+	tiupVersions := &v0manifest.VersionManifest{
+		Description: tiup.Desc,
+		Versions: []v0manifest.VersionInfo{
+			{
+				Version: v0manifest.Version(tiupver.NewTiUPVersion().SemVer()),
+				Date:    time.Now().Format(time.RFC3339),
+				Entry:   "tiup",
+			},
+		},
+	}
+	for _, goos := range []string{"linux", "darwin"} {
+		for _, goarch := range []string{"amd64", "arm64"} {
```

Comment:
I think we don't need arm/OS X

Reply:
Yes, the darwin/arm64 won't be added to manifest because there is no tarball.